tbx/project_styleguide/templates/patterns/atoms/sprites/sprites.html

```diff
@@ -35,8 +35,8 @@
         <path fill="currentColor" d="M5.923.6v14.473l4.442-4.199.635.528L5.5 16.6 0 11.402l.635-.528 4.442 4.199V.6h.846Z"/>
     </symbol>
 
-    <symbol id="chevron" viewBox="0 0 11 6" fill="none">
-        <path fill="currentColor" d="M6.02 4.297 10.06.22l.576.513-5 5.05-5-5.05.577-.513 4.039 4.078h.769Z"/>
+    <symbol id="chevron" viewBox="0 0 13 23" fill="none">
+        <path d="M1 1L11.6066 11.6066L1 22.2132" stroke="currentColor"/>
```

Comment:
Question: Will this change have any impact on other components that use this icon? e.g. the dynamic hero block?

Reply:
Yes, it does! We discussed this with Ben E., and he's keen on updating all such components